server/admin_management/serverstart.py

```diff
@@ -0,0 +1,8 @@
+# Only for local development. You can debug this file and have the server
+# running in debug mode. Not packaged for production.
+
+import uvicorn
+
+import api.app.main as main
+
+uvicorn.run(main.app, host="127.0.0.1", port=8080, log_level="info")
```

Comment:
Should we use some other port? **8080** is ok, but it is very often used by other apps or tools as default port that very often would result a conflict.

Reply:
sure, do you have a preference? I just use 8080 that is different than our backend, I'm fine to change it.

Reply:
8001 :)

Reply:
will use 8001